src/main/java/com/server/HTTPServerManager.java

```diff
@@ -25,14 +30,18 @@ public void runServer() throws IOException {
             RequestReader requestHeaderReader = new RequestReader(in);
             RequestHeaderParser requestHeaderParser = new RequestHeaderParser(requestHeaderReader.getHeader(), serverConfig.getDirectory());
 
-            PrintWriter out = openOutputStream(clientSocket);
-
             RequestParams requestParams = requestHeaderParser.getRequestParams();
 
             ResponseBuilder responseBuilder = new ResponseBuilder(requestRouter);
 
-
-            out.println(responseBuilder.getResponse(requestParams));
+            ByteArrayOutputStream baos = new ByteArrayOutputStream();
+            baos.write(responseBuilder.getResponse(requestParams));
+            byte[] result = baos.toByteArray();
+            BufferedOutputStream outputStream = new BufferedOutputStream(clientSocket.getOutputStream());
```

Comment:
I think this replaces the method on like `:67` `openOutputStream`. Maybe we can move this into that method or just delete it?